backend/remote-state/backend.go

```diff
@@ -46,6 +47,14 @@ func (b *Backend) Configure(rc *terraform.ResourceConfig) error {
 	return b.Backend.Configure(rc)
 }
 
+func (b *Backend) States() ([]string, string, error) {
+	return []string{backend.DefaultStateName}, backend.DefaultStateName, nil
```

Comment:
As per the comments above: this is where I'd move the "multiple states not supported" errors.